pkg/handlers/app.go

```diff
@@ -55,6 +55,20 @@ type SendCustomAppMetricsRequest struct {
 
 type CustomAppMetricsData map[string]interface{}
 
+type SendAppInstanceTagsRequest struct {
+	Data AppInstanceTagsData `json:"data"`
+}
+
+type AppInstanceTagsData map[string]interface{}
```

Comment:
This is `interface{}` even though tag values only support `string` because we want the handler to accept liberally and send a 400 Bad Request back to the client.

Reply:
you can handle that using `map[string]string` too.  binding will fail if data is not valid

Reply:
Done.